resources/knative-provisioner-natss/charts/knative-provisioner-natss/templates/pre-upgrade-job.yaml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Release.Name }}-pre-upgrade
+  annotations:
+    "helm.sh/hook": "pre-upgrade"
```

Comment:
That's JSON notation. YAML has no quotes.

Reply:
True that hence removed except the weights where we wanna force them to be strings.